force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -291,7 +332,8 @@ public with sharing class VariablesDataTypesOperators {
      * @return A Decimal representing the monthly paycheck.
      */
     public static Decimal calculateMonthlyPaycheck(Decimal hourlyRate, Double numberOfHours) {
-        return null; // Replace null with the variable you used to store the result
+        Decimal monthlyPaycheck = hourlyRate * numberOfHours * 4;
```

Comment:
Try creating a variable for the number 4 so it's not a magic number. https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=In%20computer%20programming%2C%20a%20magic,replaced%20with%20a%20named%20constant